servers/graphql-kotlin-server/src/main/kotlin/com/expediagroup/graphql/server/extensions/responseExtensions.kt

```diff
@@ -36,12 +35,20 @@ fun ExecutionResult.toGraphQLResponse(): GraphQLResponse<*> {
 /**
  * Convert the graphql-java type to the common serializable type [GraphQLServerError]
  */
-fun GraphQLJavaError.toGraphQLKotlinType() = GraphQLServerError(
-    message.orEmpty(),
-    locations?.map { it.toGraphQLKotlinType() },
-    path,
-    extensions
-)
+fun graphql.GraphQLError.toGraphQLKotlinType(): GraphQLServerError {
+    val locations = this.locations?.map { it.toGraphQLKotlinType() }
+    val newExtensions = this.extensions?.toMutableMap() ?: mutableMapOf()
```

Comment:
if extensions are null and there is no error type information I'd assume the `newExtensions` should be null as well

Reply:
Not if we are going to immediately set the `classification` anyway

Reply:
I suppose now that we are no longer constructing a custom `GraphQLError` we can just use the extensions as is